src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  padding: 0 40px 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  align-self: center;
+  margin: 10px 0 10px 50px;
+}
+
+.nav__lists {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  color: black;
+  position: relative;
+  display: block;
+  font-style: normal;
+  padding: 23px 0;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-right: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children

Hint:
Last child shouldn't have margin. Use `:not(last-child)`